sourcemash/frontend/templates/base.html

```diff
@@ -30,18 +30,17 @@
 
   <body>
 
-    <div class="container">
-      {% block layout %}{% endblock %}
-    </div>
-
     <!--Import jQuery before materialize.js-->
-    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
     {% assets "js_vendor" %}
       <script src="{{ ASSET_URL }}"></script>
     {% endassets %}
     {% assets "js_default" %}
       <script src="{{ ASSET_URL }}"></script>
     {% endassets %}
+    
+    <div class="container">
+      {% block layout %}{% endblock %}
+    </div>
```

Comment:
Can we try loading all backbone files in base.html? It would be nice to get all js files loaded after loading the html so that loading doesn't take too long.